### ISSUE TITLE
List with all bots ranked by number of followers

### ISSUE THREAD

**Issue by t-pfaff** [codeforamerica/CutePets#35]
In addition to the where.geojson a list with all bots ranked by the number of followers would be great. Redeploying the Twitter bot is not too difficult, but maybe getting a decent number of followers is. That way people who redeploy the bot can see which bot is doing best and can ask the people behind it for their follower acquisition strategy.
Makes sense?

**Comment by boonrs** [comment]
I like the idea of sharing knowledge on how to increase followers. But I think the metric of number of followers is deceiving. For example, a small town may have a lot of followers proportional to it's population but ranked low.

We could ask existing implementations to contribute their knowledge to the wiki. That would institutionalize the knowledge for others.

With that being said, I think a follower count would be interesting with the added context of city size.

**Comment by t-pfaff** [comment]
You are right, it only makes sense as a relative number.
Who could take this forward? (adding knowledge to the wiki + relative follower count)

**Comment by boonrs** [comment]
As a starting point, we could ping the twitter bots on the map for tips.

**Comment by drewrwilson** [comment]
I like the ideas here! Good to talk about how to make this more impactful. I think more followers is good for these bots, but I don't know if follower count (even if adjusted for population of a given city) is the best measure of success for this project. More about that in a sec, but first about that increasing follower count: @boonrs, great idea to hit up other folks for thoughts on growing followers and @t-pfaff putting some thoughts in the wiki is a great idea too. 

Personally, I think this impact of this project is more related to making open data useful than it is about helping animals. Some ideas for how to measure the success of this project: How many people feel empowered and excited about open data after redeploying this? How many people now have experience working with open data after redeploying this project? How many people deployed their first app because of this? Have we been able to use this small project as a way to show local gov reps that we can do fun, creative things when they open up data for us? Have we been able to use this project to persuade people in power to open more data and become more transparent?

**Comment by drewrwilson** [comment]
:speech_balloon: :speech_balloon: :speech_balloon: :speech_balloon:

**Comment by t-pfaff** [comment]
I totally agree that (relative) followers are not the ideal impact measure. But it's a first proxy that is simple to build.

I set up a first wiki page, asking for both impact stories and open data impact stories. Please check and edit.

For the tweet something like: `@XYZ So cool you deployed this bot! We'd like to see more bots. Please share some tips on the wiki: https://github.com/codeforamerica/CutePets/wiki`_ Do you want to do that from your accounts? Otherwise (maybe with less credibility) our intern can do that from our account. Let me know!

Our intern can also prepare a table with the follower numbers relative to the population. Would be great to have that with dynamic numbers from the Twitter API. Where can this be hosted? If you guys don't want to do it, we could do it on our Wordpress blog (we recently did a dynamic table fetching data from the meetup.com API - see [here](http://blog.datalook.io/definitive-guide-data-science-good/)).

**Comment by boonrs** [comment]
We'd :heart: if your intern wanted to take this on! If they find they're not getting credibility with the request, we can help out. If you're willing to host it on your blog, that would be :smiley: for us. Otherwise, CfA has resources we can use.

:watermelon: Thank you for your thoughtfulness and help. :tada:

**Comment by t-pfaff** [comment]
Cool, will ask Costantin to work on that in the next couple of days.

**Comment by t-pfaff** [comment]
Which list is more complete, [where.geojson](https://github.com/codeforamerica/CutePets/blob/master/where.geojson) or Drew's [Twitter list](https://twitter.com/drewSaysGoVeg/lists/cutepetseverywhere/members)?

**Comment by boonrs** [comment]
I believe the where.geojson is more complete as it's crowd sourced.

**Comment by CostantinB** [comment]
I’ve took all the handlers from where.geojson and from the twitter list and removed the duplicates. The table imports followers automatically (it refresh ~ every 30 minutes) feel free to add handlers in the first column, it should do all the rest automatically, the table is in our cute-pets twitter bot folder in docs.datalook.io

enjoy! :)

**Comment by boonrs** [comment]
Lovely @CostantinB!  I'll put the link in the Readme to share the knowledge.

**Comment by techieshark** [comment]
I just got the ping for the Mesa bot.

**Comment by t-pfaff** [comment]
@boonrs We tweeted to all bots but didn't get any answer (apart from @techieshark in this thread). And no one has contributed to the wiki page so far. Maybe we didn't have enough credibility. Can you help out?

**Comment by boonrs** [comment]
Yes, I'd be happy to help out @t-pfaff . A hypothesis as to why no one has replied, perhaps whoever deployed the bot doesn't check the twitter feed. Might be more productive to hit up the github profile of whoever created a pull request to add the cities.

**Comment by t-pfaff** [comment]
Yes, makes sense. Let us try to do that and if it still doesn't work, you could maybe ping them from your account. @CostantinB Can you please go through the [commits]() of `where.geojson` and ping the guys who added cities?
@boonrs: Some cities where added by you and @drewrwilson. Where would you look for the people behind these bots?
